ngraph/frontend/onnx/frontend/src/op/matmul_integer.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/matmul_integer.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "dequantize_linear.hpp"
+#include "matmul.hpp"
+#include "ngraph/opsets/opset6.hpp"
+#include "quantize_linear.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector matmul_integer(const Node& node) {
+    const OutputVector& inputs = node.get_ng_inputs();
+
+    const auto& A = inputs.at(0);
+    const auto& B = inputs.at(1);
+    const auto& A_zero_point = (inputs.size() > 2) ? inputs.at(2) : ngraph::op::Constant::create(ngraph::element::i64, {1}, {0});
+    const auto& B_zero_point = (inputs.size() > 3) ? inputs.at(3) : ngraph::op::Constant::create(ngraph::element::i64, {1}, {0});
+
+    const auto& converted_A = std::make_shared<opset6::Convert>(A, element::i32);
+    const auto& converted_B = std::make_shared<opset6::Convert>(B, element::i32);
+
+    const auto& converted_A_zero_point = std::make_shared<opset6::Convert>(A_zero_point, element::i32);
```

Comment:
Please use `default_opset` unless there is a reason to avoid it.

Reply:
Updated code to use `default_opset`, thank you!